homeassistant/components/nut/const.py

```diff
@@ -209,29 +215,32 @@
         key="ups.start.auto",
         name="Start on AC",
         icon="mdi:information-outline",
+        entity_category=ENTITY_CATEGORY_CONFIG,
     ),
     "ups.start.battery": SensorEntityDescription(
         key="ups.start.battery",
         name="Start on Battery",
         icon="mdi:information-outline",
+        entity_category=ENTITY_CATEGORY_CONFIG,
     ),
     "ups.start.reboot": SensorEntityDescription(
         key="ups.start.reboot",
         name="Reboot on Battery",
         icon="mdi:information-outline",
+        entity_category=ENTITY_CATEGORY_CONFIG,
     ),
     "ups.shutdown": SensorEntityDescription(
         key="ups.shutdown",
         name="Shutdown Ability",
         icon="mdi:information-outline",
+        entity_category=ENTITY_CATEGORY_CONFIG,
     ),
     "battery.charge": SensorEntityDescription(
         key="battery.charge",
         name="Battery Charge",
         native_unit_of_measurement=PERCENTAGE,
         device_class=DEVICE_CLASS_BATTERY,
         state_class=STATE_CLASS_MEASUREMENT,
-        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
```

Comment:
Why did you remove this one?

Reply:
That was because I think for a UPS specifically the battery charge isn't diagnostic but rather a primary entity you want to have. It is what it provides to your environment. For other devices you care about a switch/binary sensor etc and "oh by the way" there's a battery but not important. Whereas for a UPS you want to know your battery's charge. Especially since if I'm not mistaken now diagnostic/config entities don't show on auto-generated UI, so also thought a user would want this one to show up there.

Reply:
Ahh, ok, makes sense then!